README.md

```diff
@@ -0,0 +1,35 @@
+# @sveltejs/branding
+
+Logos and Logotypes for Svelte in different formats.
+
+`svelte-logo.pdf` or `sapper-logo.pdf` are editable Adobe-Illustrator-CC-files which can be useful for occasions where none of the provided solutions fit.
+
+`colors.ase`  is the exported svelte color-palette [Adobe CC]
+
+**Don't** alter paths or proportions in any way.
+
+### Icons
+
+* Icons are available at sizes 16, 24, 32, 48, 64 or 128px
+* Either 1x or 4x resolution
+* .png or .svg
+
+![svelte icon 32x32](https://github.com/sveltejs/branding/icons/svelte-32.png)
```

Comment:
I think it should be `![svelte icon 32x32](./icons/svelte-32.png)` so that it works independently from the branch we're in.